requirements.txt

```diff
@@ -0,0 +1,3 @@
+wheel
```

Comment:
Why ?

Reply:
Because I got wheel not found on github action, don't known why

Reply:
I think this is because it's in `setup_requires`, which I doubt you need if this is run straight from the repo anyways

Reply:
Indeed, everything works fine without it :)